test/test19.sh

```diff
@@ -0,0 +1,10 @@
+printf "build/marc --output=test.tmp input/test19.opl\n";
+
+export LD_LIBRARY_PATH=build/
+build/alex --output=test.tmp input/test19.opl
```

Comment:
Please save output to tmp/test19.lex. This way the Makefile target "clean" will get rid of the test output files.